spec/components/selection/Switch.spec.tsx

```diff
@@ -0,0 +1,86 @@
+import { mount, shallow } from 'enzyme';
+import * as React from 'react';
+import { Switch } from '../../../src/components';
+import SelectionTypes from '../../../src/components/selection/SelectionTypes';
+import { SelectionInput } from '../../../src/components/selection/SelectionInput';
+import SelectionStatus from '../../../src/components/selection/SelectionStatus';
+
+describe('Switch Component', () => {
+  describe('Switch component test suite => ', () => {
+    afterEach(() => {
+      jest.restoreAllMocks();
+    });
+
+    it('render with default props and initial value', () => {
+      const wrapper = shallow(
+        <Switch name="default-checkbox-name" value="default-checkbox-value" />
+      );
+      expect(wrapper.length).toEqual(1);
+      expect(wrapper.find(SelectionInput).length).toBe(1);
+      expect(wrapper.find(SelectionInput).prop('type')).toEqual(
+        SelectionTypes.SWITCH
+      );
+      expect(wrapper.find(SelectionInput).prop('name')).toEqual(
+        'default-checkbox-name'
+      );
+      expect(wrapper.find(SelectionInput).prop('value')).toEqual(
+        'default-checkbox-value'
+      );
+    });
+
+    it('with "checked" status', () => {
+      const onChangeCallback = jest.fn();
+      const wrapper = mount(
+        <Switch
+          name="checked-state-checkbox-name"
+          value="checked-state-checkbox-value"
+          status={SelectionStatus.CHECKED}
```

Comment:
I suggest to use another status, maybe 'SelectionStatus.SWITCHED' ?

Reply:
Thank you for your review @symphony-jean-michael :)

Indeed, I forgot to adapt the test file. I removed the 'mixed status' test, and replaced the term 'checkbox' by 'switch'.

For the CHECKED status, if we use SWITCHED, we will have to adapt SelectionInput, and also adapt the classnames in the sass mixins.

Or we can avoid theses changes by having: 
```
enum SelectionStatus {
  CHECKED = 'checked',
  SWICHED = 'checked',
  MIXED = 'mixed',
  UNCHECKED = 'unchecked',
  UNSWITCHED = 'unchecked',
}
```